cylc/flow/cfgspec/suite.py

```diff
@@ -841,11 +841,76 @@
 
                 ``$CYLC_TASK_CYCLE_POINT/shared/``
             ''')
-            Conf('execution polling intervals', VDR.V_INTERVAL_LIST, None)
-            Conf('execution retry delays', VDR.V_INTERVAL_LIST, None)
-            Conf('execution time limit', VDR.V_INTERVAL)
-            Conf('submission polling intervals', VDR.V_INTERVAL_LIST, None)
-            Conf('submission retry delays', VDR.V_INTERVAL_LIST, None)
```

Comment:
These were undocumented because they are deprecated by the `[platforms]` configurations in the global config.

I think they should probably look something like this:

```rst
.. warning::

   Deprecated, use :cylc:conf:`global.cylc[platforms][<platform name>]submission polling intervals`
```

Reply:
I didn't think that these were deprecated, but were an over-ride mechanism.

Reply:
Just took a look at the [config proposal](https://cylc.github.io/cylc-admin/proposal-config-changes.html), we didn't write anything down about deprecating them (helpful). From what I can remember of the Wellington discussion we couldn't come up with any logical reason for setting "* polling intervals" or "submission retry delays" on a per-task basis. I.E. submission things should be platform related, execution things should be task related.

Really sorry @wxtim, I had a daft moment back there and missed that this list includes two execution configs. The "execution time limit" and "execution retry delays" settings are definitely still valid (there's no other way of defining them). They should be documented here, MB, sorry.

Reply:
These things happen. I'm amazed that you keep as much in your brain as you do.
I'm glad I argued.
I'll put the execution time limit/retry delay docs back. 
Should I formally deprecate the submission settings or keep the doc links?

Reply:
I'm not sure if we can deprecate because the config has moved to another config. Should probably just raise a warning in the upgrader function somewhere.

Reply:
I think that I've fixed this for now. I've put a custom deprecate on the items moved to global.